buildpipeline/DotNet-CoreFx-Trusted-Linux.json

```diff
@@ -511,7 +511,7 @@
   ],
   "buildNumberFormat": "$(date:yyyyMMdd)$(rev:-rr)",
   "jobAuthorizationScope": "projectCollection",
-  "jobTimeoutInMinutes": 60,
+  "jobTimeoutInMinutes": 90,
```

Comment:
I assume the timeout isn't related to the signing changes at all but just a general timeout issue? If so I would like to make sure we have an issue tracking the investigation of this timeout.

Reply:
Sure, I can open one;  basically ever since recent CPU bug patching our builds have crept up in time.